index.html

```diff
@@ -1,7 +1,26 @@
 <!DOCTYPE html>
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Gray Button on Click</title>
+    <style>
+        button {
+            padding: 10px 20px;
+            background-color: darkblue;
+            color: #fff;
+            border: none;
+            cursor: pointer;
+        }
+
+        .gray-button {
+            background-color: #ccc;
+        }
+    </style>
+</head>
 <html>
   <body>
-    <button id="btn">play</button>
+    <button id="play" onclick="playMusic()">play</button>
```

Comment:
I wonder what's the difference between setting onclick here vs setting it in the script itself.  I noticed you had to do window.playMusic = playMusic, etc... maybe it's best to leave it in the script?